src/pfe/file-watcher/server/src/controllers/projectsController.ts

```diff
@@ -1055,6 +1075,11 @@ export async function updateProjectInfo(projectID: string, keyValuePair: UpdateP
         keyValuePair.saveIntoJsonFile = true;
     }
 
+    // remove case
+    if (!keyValuePair.value) {
```

Comment:
If `keyValuePair.value` is falsy could this lead to removing something form the projectInfo unintentionally?

Reply:
good point here, I'll change it

Reply:
Fixed by https://github.com/eclipse/codewind/pull/1535/commits/2b2cd19689c4425d9144e44663ff603491608857